generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -183,6 +183,14 @@ jhipster:
                 # Token is valid 24 hours
                 tokenValidityInSeconds: 86400
     <%_ } _%>
+    <%_ if ((applicationType === 'microservice' || applicationType === 'uaa') && authenticationType === 'uaa') { _%>
+        clientAuthorization:
+            # change this depending on your authorization server
+            tokenUrl: http://uaa:9999/oauth/token
```

Comment:
this would add a new question to microservice generator, similar to gateway...but yes, it could be done